lib/notesclub_web/live/notebook_live/share_component.ex

```diff
@@ -0,0 +1,28 @@
+defmodule NotesclubWeb.NotebookLive.ShareComponent do
+  @moduledoc """
+  Icon to share to X/Twitter
+  """
+  use Phoenix.Component
+
+  attr :share_to_x_text, :string, required: true
+
+  @doc """
+  Share to X/Twitter icon
+  """
+  def share_to_x(assigns) do
+    ~H"""
+    <.link href={"https://twitter.com/intent/tweet?text=#{@share_to_x_text}"} target="blank_">
+      <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 512 512">
```

Comment:
Let's increase it a bit so the icon size is like GitHub's icon:
```suggestion
      <svg xmlns="http://www.w3.org/2000/svg" width="35" height="35" viewBox="0 0 512 512">
```